src/test/java/uk/gov/cshr/controller/emailUpdate/AgencyTokenVerificationControllerTest.java

```diff
@@ -45,23 +40,22 @@
 @SpringBootTest
 public class AgencyTokenVerificationControllerTest {
 
```

Comment:
Add a test method for the default scenario in the try block

Reply:
There are no tests for this scenario as the `DEFAULT` block of the switch is currently unreachable as I'm passing in an enum. The reason I've done this is to future proof/guard the switch against any enums being misused. i.e. if a new enum is created and is used as part of this verification flow, we can throw an expection.

I had a look at a variety of alternatives. Introducing a third enum (e.g. value = UNKNOWN) could work, but this would create a new problem where we have to pass a null email parameter when returning the `VerificationCodeDetermination` object.

From a testing perspective I had a look at using powermock to mock the enum, but this would then cause problems with using mockMVC in the rest of the class.

I've considered this, but ultimately don't think the alternatives are worth pursuing in order to reach 100% coverage.